### ISSUE TITLE
add streams via config file

### ISSUE THREAD

**Issue by routerino** [iEvgeny/cctv-viewer#10]
Hi,

This might be a dumb question, but rather than using the GUI (which is handy, but I generally only have terminal access) is there a place to modify or create a config?

**Comment by iEvgeny** [comment]
Hi!
Of course. All settings are stored in the configuration file: "~/.config/CCTV Viewer/CCTV Viewer.conf"
After modify the configuration file, you must restart the application.

**Comment by routerino** [comment]
Thanks, had a look. Looks like a minified json string that is then double quoted with many, many escaped quotes.

It's probably too much to ask, but a future suggestion is separating the config into a formatted JSON file. Would make parsing it soooo much easier with a tool like `jq`.

**Comment by wiryonolau** [comment]
Hi when is the config file generated ? This path doesn't exist ~/.config/CCTV Viewer/CCTV Viewer.conf
I'm using non snap version

**Comment by Billysredhat** [comment]
I have found that I have to start cctv-viewer with the default settings and then close it for the config file path to be created if it is a new install. Also it could just be a typo in your question, but the exact path is "~/.config/CCTV\ Viewer/CCTV\ Viewer.conf" without the quotes.

**Comment by wiryonolau** [comment]
Yes it need to be escape. But it actually not there so I have to find in .config and found out it under T171RU

```
~/.config/T171RU/CCTV\ Viewer.conf
```

I try to recreate and it still there. What is T171RU ?

**Comment by wiryonolau** [comment]
Suggestion to move config format to 100% json, using ini and json together is confusing

**Comment by 3LD** [comment]
![Image](https://github.com/user-attachments/assets/b4da68d5-1ff2-4385-af22-e43e6c155fed)
